apollo-federation/src/sources/connect/expand/mod.rs

```diff
@@ -514,6 +513,10 @@ mod helpers {
                         }
                     }
 
+                    Parameter::Context { item: _, paths: _ } => {
+                        // TODO Implement $context handling
+                    }
```

Comment:
@dylan-apollo @nicholascioli Here's where I think we need to add support for generating the `$context` JSON object, like we do with `Parameter::Argument` and `Parameter::Sibling` variables.